frontend/src/views/SQR/Form.vue

```diff
@@ -312,6 +310,8 @@ export default class Form extends Vue {
   };
   submitSuccess = false;
   formHasError = false;
+  isCoordinator = false;
+  allotmentError = "";
```

Comment:
As this property is used not only for the allotment errors, I'd name it more generic. For example,
```suggestion
  errorMessage = "";
```